src/barfTurn.ts

```diff
@@ -64,7 +64,15 @@ import {
   waterBreathingEquipment,
 } from "./outfit";
 import postCombatActions from "./post";
-import { determineDraggableZoneAndEnsureAccess, digitizedMonstersRemaining } from "./wanderer";
+import {
+  determineDraggableZoneAndEnsureAccess,
+  digitizedMonstersRemaining,
+  DraggableFight,
+} from "./wanderer";
+
+const sober = () => myInebriety() <= inebrietyLimit();
```

Comment:
Can this run while Stooper is active?

Reply:
I believe that we unequip the stooper when we run diet code, and that we'll always run diet code by the time we go here. but that is worth special casing regardless.